lib/App/Rakubrew.pm

```diff
@@ -311,11 +311,15 @@ EOL
 
     } elsif ($arg eq 'build-zef') {
         my $version = get_version();
+	my $zef_version = shift(@args);
```

Comment:
Can you replace all tabs with spaces?

Reply:
My emacs is playing games on me... Done now.